src/domains/chart/utils/get-portal-node-styles.ts

```diff
@@ -80,6 +82,13 @@ export const getPortalNodeStyles: GetPortalNodeStyles = (
     height = heightFromLocalStorage.replace(/"/g, "")
   }
 
+  if (shouldAddSpecialHeight) {
+    const heightOverriden = isLegendOnBottom
+      ? window.innerHeight * 0.5
+      : window.innerHeight * 0.4
+    height = `${heightOverriden}px`
```

Comment:
Is this also working in the virtualized dashboard?

Reply:
Yep, i checked it both in Agent Dashboard and Cloud's Node-View, it has impact only on initial mount, i put more info on the other comment